server/routes/incidentDetails/incidentDetailsPage.ts

```diff
@@ -297,6 +297,9 @@ const extractIncidentDetails = (readDraftIncidentDetails: ExistingDraftIncidentD
 
   if (formatDate(readDraftIncidentDetails.dateTime) === formatDate(readDraftIncidentDetails.dateTimeOfDiscovery)) {
     radioValue = 'Yes'
+    /* eslint-disable no-param-reassign */
+    readDraftIncidentDetails.dateTimeOfDiscovery = null
+    /* eslint-enable no-param-reassign */
```

Comment:
We try to not use reassignments where possible in the codebase as it can lead to confusion down the line, especially in a file like this which is already really quite confusing!
It would be better if you were to create a function to determine what radioValue was here. 
e.g. 
const getRadioValue = (dateTime, dateOfDiscovery) => {
if (dateTime === dateOfDiscovery) {
return 'Yes'
} 
return 'No'
}

In terms of this `readDraftIncidentDetails.dateTimeOfDiscovery = null`
Again I'd try not to reassign anything where we don't desperately need to. Instead I think we could use a conditional in the return like
const discoveryDate = dateTime === dateOfDiscovery ? null : readDraftIncidentDetails.dateTimeOfDiscovery

Reply:
replaced with sugested